.eslintrc

```diff
@@ -130,7 +130,7 @@
         // "no-duplicate-imports": ["error", { "includeExports": true }],
         // should be error, no?
         "@typescript-eslint/no-unused-vars": ["warn", { "argsIgnorePattern": "^_" }],
-        "@typescript-eslint/consistent-type-imports": ["error", { "disallowTypeAnnotations": false }],
+        "@typescript-eslint/consistent-type-imports": ["error", { "disallowTypeAnnotations": false, "fixStyle": "separate-type-imports" }],

```

Comment:
If we do not have it as warning or errors, should you call fix in pre commit tasks ?

Reply:
that would take a long time for each commit, AND will not solve the issue if it's already inlined.